omc3/plotting/plot_amplitude_detuning.py

```diff
@@ -521,9 +526,9 @@ def _plot_detuning_3d(ax: Axes, data: Dict[str, AmpDetData], label: str, color=N
 
     # plot errorbars
     for idx in np.arange(0, len(tune)):
-        x, dx = jx[idx], jx_err[idx]
-        y, dy = jy[idx], jy_err[idx]
-        z, dz = tune[idx], tune_err[idx]
+        x, dx = jx.iloc[idx], jx_err.iloc[idx]
+        y, dy = jy.iloc[idx], jy_err.iloc[idx]
+        z, dz = tune.iloc[idx], tune_err.iloc[idx]
```

Comment:
This fixes the `FutureWarning` for using `Series.__getitem__` which is deprecated.